extension/writing-process/src/background.js

```diff
@@ -9,7 +9,7 @@ var RAW_DEBUG = false;
 /* This variable must be manually updated to specify the server that
  * the data will be sent to.  
 */
-var WEBSOCKET_SERVER_URL = "wss://learning-observer.org/wsapi/in/" 
+var WEBSOCKET_SERVER_URL = "wss://learning-observer.org/wsapi/in?source=extension" 
```

Comment:
We really ought to have the url not hardcoded and pulled from a config of some sort, but that's way beyond the scope of this PR and the work you are doing.
Mostly leaving this note to push it to the front of my brain (I'm most likely to work on it)